applications/peripheral_sensor_node/configuration/nrf54l15pdk_nrf54l15_cpuapp/app.overlay

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/{
+	/delete-node/ leds;
+
+	leds0 {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio2 9 GPIO_ACTIVE_HIGH>;
+			label = "Green LED 0";
+		};
+	};
+
+	leds1 {
+		compatible = "gpio-leds";
+		led1: led_1 {
+			gpios = <&gpio1 10 GPIO_ACTIVE_HIGH>;
+			label = "Green LED 1";
+		};
+	};
+
+	leds2 {
+		compatible = "gpio-leds";
+		led2: led_2 {
+			gpios = <&gpio2 7 GPIO_ACTIVE_HIGH>;
+			label = "Green LED 2";
+		};
+	};
+
+	leds3 {
+		compatible = "gpio-leds";
+		led3: led_3 {
+			gpios = <&gpio1 14 GPIO_ACTIVE_HIGH>;
```

Comment:
On pca63565 GPIO P1.14 is/may be used as interrupt signal from ADXL362 (requires jumper on header P3)
https://github.com/nrfconnect/sdk-nrf/blob/main/boards/shields/pca63565/boards/nrf54l15pdk_nrf54l15_cpuapp.overlay#L88

Reply:
Right. Added note to documentation.